### ISSUE TITLE
[Bug Report] Super reacts have a nested element with extra border radius

### ISSUE THREAD

**Issue by NeoNyaa** [ClearVision/ClearVision-v6#218]
### Did You Search Other Issues?

- [X] I have checked other issues

### Does This Bug Show On The Default Theme?

- [X] I have reproduced this on the default theme

### Describe The Bug

Super reacts have this weird inner element that has some rounding which isnt in line with the theme.

This is what it looks like as of now
![image](https://github.com/user-attachments/assets/9ce0b280-05fe-45f4-9da7-67cd426136f8)

This is what it looks like with the following tweak:
```css
.burstGlow_ec6b19 {
  /* I chose 3px as it is inline with the rest of the theme and also adds a kind of feathering effect to the border which would otherwise not be present when set to 0px
  
  Basically what I am saying is either set it to 3px for that VERY subtle feathing effect or set to 0 if you really dont care about that. */
  border-radius: 3px;

}
```
![image](https://github.com/user-attachments/assets/a1b24ee1-7055-4507-8fd1-7a11f54f9dbe)

### Reproduction Steps

```bash
Just use the base theme and look for a super react and look really closely, you should see it.
```

### Screenshots

```bash
I added screenshots to the describe the bug section
```

**Comment by Leozard** [comment]
will be fixed in the next update, holy it's so hidden

**Comment by NeoNyaa** [comment]
> will be fixed in the next update, holy it's so hidden

I know right, good to know it's going to be fixed. Let me know if I should close this issue or close it yourself if that's preffered.
